### DISCUSSION TITLE
Sound Output Issues on MacOS

### DISCUSSION THREAD

**Discussion by bombarde32f** [GrandOrgue/grandorgue#1167] [Help]
I have my MacBook Pro (Mid 2014, 3 GHz Dual-Intel Core i7) connected via an M-Audio MIDI interface to my Allen Renaissance 230 organ. The MBP is running macOS Big Sur 11.6.8. The MIDI interface (an M-Audio Uno USB) works fine with all other applications including DAWs and notation software. However, when launching Grand Orgue, I keep getting the error "Output device Core Audio (PA): Ausgang (integriert) not found - no sound output will occur." Not sure why this is a problem in the more recent version of Grand Orgue (v3.7.0-1). Very frustrating.
<img width="416" alt="Screen Shot 2022-08-27 at 7 49 40 PM" src="https://user-images.githubusercontent.com/64907344/187053037-65206426-b6c9-4337-a7e6-72be4aff8736.png">

**Comment by rousseldenis** [comment]
> I have my MacBook Pro (Mid 2014, 3 GHz Dual-Intel Core i7) connected via an M-Audio MIDI interface to my Allen Renaissance 230 organ. The MBP is running macOS Big Sur 11.6.8. The MIDI interface (an M-Audio Uno USB) works fine with all other applications including DAWs and notation software. However, when launching Grand Orgue, I keep getting the error "Output device Core Audio (PA): Ausgang (integriert) not found - no sound output will occur." Not sure why this is a problem in the more recent version of Grand Orgue (v3.7.0-1). Very frustrating.
> <img width="416" alt="Screen Shot 2022-08-27 at 7 49 40 PM" src="https://user-images.githubusercontent.com/64907344/187053037-65206426-b6c9-4337-a7e6-72be4aff8736.png">
> 

Hi.

I can have the same issue if you start directly GrandOrgue after computer startup (I haven't investigated more) but if you validate, GrandOrgue loads fine and sound is ok( I'm on Ubuntu). Can you confirm ?

**Comment by willeke1234** [comment]
Did you have an older version of GrandOrgue installed? Are the system language settings still the same?

**Reply by oleg68** [reply]
@bombarde32f please go to the Files->Settings->Audio and try to change the audio device. What can you see?

**Reply by bombarde32f** [reply]
> Did you have an older version of GrandOrgue installed? Are the system language settings still the same?

I have v3.7.0-1, which looks to be the most recent version. My system language is English.

**Reply by bombarde32f** [reply]
> @bombarde32f please go to the Files->Settings->Audio and try to change the audio device. What can you see?

![image](https://user-images.githubusercontent.com/64907344/187099974-e371af5f-5122-4cd7-bdc2-fcc2d8678098.png)

**Reply by bombarde32f** [reply]
> @bombarde32f please go to the Files->Settings->Audio and try to change the audio device. What can you see?

I had to select the correct audio device. That fixed it! Thank you so much!
